src/widgetastic_patternfly/__init__.py

```diff
@@ -2351,3 +2359,219 @@ def item_select(self, item, close=True):
         finally:
             if close:
                 self.close()
+
+
+class SparkLineChart(Widget, ClickableMixin):
+    """Represents the Spark Line Chart from Patternfly (Data Visualization).
+
```

Comment:
Providing urls to the Patternfly page for each widget could be helpful.

Reply:
I like this suggestion :100: . Its in my pocket... I want to improve `code-coverage` of `wt.pfy`. I am planning to spend day and push all test for all `widgets`. At same time I will add this for all `widgets`.